hil/ext/switches/brocade.py

```diff
@@ -280,6 +286,89 @@ def _disable_native_tag(self, interface):
         url = self._construct_url(interface, suffix='trunk/tag/native-vlan')
         self._make_request('DELETE', url, acceptable_error_codes=(404,))
 
+    def save_running_config(self):
+        url = self.hostname + '/rest/' + 'operational-state/bna-config-cmd'
+        body = '<bna-config-cmd><src>running-config</src><dest>startup-config \
+                </dest></bna-config-cmd>'
+        response = self._make_request('POST', url, data=body)
+
+        # check if it succesfully completed and log it
+        tag = '{urn:brocade.com:mgmt:brocade-ras}session-id'
+        root = etree.fromstring(response.text)
+        session_id = root.find(tag).text
+        if self._check_bna_config_status(session_id):
+            logger.debug('Saved running config to startup config')
+        else:
+            logger.debug('Configuration not saved in reasonable time')
```

Comment:
I think this should be logged at error level; if the config isn't saving correctly that's something a sysadmin should respond to.